benchmarks/tpcds/prepare.py

```diff
@@ -66,6 +72,9 @@ def _ingest_task_impl(self, table, dbgen):
         return task
 
     def get_ingest_tasks(self, table):
+        if self.args.data_dir:
```

Comment:
Maybe this has to be moved under line #79

Reply:
no it shouldnt as the csv files are separate but the commands for non-csv files are not... :(

Reply:
No. The `_returns` special case is there because when generating the data on the fly, sales and returns tables are created together. But if a data dir is used, all the files are already created, so you shouldn't skip returns tables.

Reply:
if data_dir is used only for the csv files, how about rename it to csv_dir?